src/scanner.c

```diff
@@ -1002,10 +1002,17 @@ char buf[40];
   strcat( buf, coef ); 
   sscanf( buf, "%lf", &val );
 
+  /*  switch( type ) {
+      case LOSS_FAM:*/
   Stoich_Right[ crtSpec ][ EqNr ] = val;
   Stoich[ crtSpec ][ EqNr ]       = val;
 
-  /*printf("\nSpecies %s with Stoich[%d][%d] = %f. SpcNr=%d. Code=%d. ReverseCode=%d",spname,crtSpec,EqNr,Stoich[crtSpec][EqNr],SpcNr,Code[crtSpec],ReverseCode[ code ]);*/
+  /*if ( type == LOSS_FAM ) {
```

Comment:
These were updates by @msl3v for the prod/loss capability of KPP in GEOS-Chem

Reply:
Do we want an error or a warning for duplicate equations? I'm not sure, but I noticed this has changed here.

Reply:
I can restore this.  I think it's good to have the warning.

Reply:
In dev, it is an error. KPP stops immediately after the first set of duplicate equations. This ensures that the problem is not overlooked.

In add-geos-chem-updates, it is a warning which may be overlooked by the user. The advantage, however, is that it will probably list all sets of duplicate equations.

Reply:
Now fixed with commit 966a6dbe74b7c1cce8029b3e5871d1829db903c0